tests/e2e/test_e2e.py

```diff
@@ -607,6 +602,35 @@ def test_e2e():
 
     assert f1 > 0.7
 
+    # Save a fonduer model
```

Comment:
It would be better to have separate tests for ml_flow.

Reply:
Add additional unit tests.